server/lib/csp.js

```diff
@@ -6,14 +6,14 @@
 // option 'csp.enabled' is set (default true in development), with a special
 // exception for the /tests/index.html path, which are the frontend unit
 // tests.
-
 'use strict';
 
 var helmet = require('helmet');
 var config = require('./configuration');
 var url = require('url');
-
+/*eslint-disable */
 var SELF = "'self'"; //jshint ignore: line
+/*eslint-enable */
```

Comment:
Actually, we can use the fancy `avoid-escape` option in the `quotes` rule for this!

Just update your .eslintrc file and add "avoid-escape" as the last parameter:
```yaml
quotes: [2, single, avoid-escape]
```
([source](http://eslint.org/docs/rules/quotes.html))